src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: gray;
+}
+
+.header {
+  background-color: white;
+  width: 1200px;
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 20px;
```

Comment:
Avoid fixing the container size as it can lead to overflow or accidental scroll bar if the content requires more space. If there is no requirement to fix the size, consider using `max-width` and `height: auto;` or other responsive design techniques.